### PR TITLE
problem with register deleted clients anew

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/uyuni-project/uyuni-docs/issues/2076
per feedback I removed this corner case trouble from CCG delete-clients.adoc
also see https://github.com/SUSE/spacewalk/issues/19385

When this is merged reference it in master and 4.3 here https://github.com/uyuni-project/uyuni-docs/pull/1982 and https://github.com/uyuni-project/uyuni-docs/pull/2086

~**don't forget to run the new file script before merging!**~ done

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by mcalmer** [comment]
Seems to be not ready for review yet.

**Comment by keichwa** [comment]
> Seems to be not ready for review yet.

Should be ready now.  I removed the draft tag.  Ignore the l10n-weblate changes (@jcayouette , can you verify those?).  The only "real" change it the added tshoot-delclients.adoc file.

File: modules/administration/pages/troubleshooting/tshoot-delclients.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+////
+// move to Trouble Shooting and link from here
+Sometimes a new registration of a deleted (unregistered) client might not be possible.
+To solve this issue, some Salt cache files should be deleted on the {productname} Server (Salt master) before trying to re-register again:
+
+----
+rm /var/cache/salt/master/thin/version
+rm /var/cache/salt/master/thin/thin.tgz
+----
```
Comment (mcalmer) [inline modules/administration/pages/troubleshooting/tshoot-delclients.adoc:31]: This seems to be commented. The file is empty. I think this is not what we want

**Review by mcalmer — CHANGES_REQUESTED** [review]

File: modules/administration/pages/troubleshooting/tshoot-delclients.adoc (hunk 1 above)
Comment (keichwa) [reply modules/administration/pages/troubleshooting/tshoot-delclients.adoc]: gosh, stupid me ;-(  Thanks for spotting!

**Review by mcalmer — APPROVED** [review]
looks good now

**Review by jcayouette — COMMENTED** [review]
Was this a manual change or did you run the script to adjust the configuration files?

**Comment by keichwa** [comment]
> Was this a manual change or did you run the script to adjust the configuration files? Those files should never be manually adjusted. Run the script within the l10 directory. If so then it should be fine.

Changes were caused by the script.  Ok, then it is ready to be merged.

**Review by 0rnela — APPROVED** [review]

**Pull request merged**